alertanio/config/resources/db_init.yaml

```diff
@@ -23,3 +24,17 @@
            zulip_subject VARCHAR(255),
            templ_id SERIAL REFERENCES templates
          )'
+- name: blackouts
+  params: 'CREATE TABLE blackouts
+         (
+           blackout_id SERIAL PRIMARY KEY,
+           environment VARCHAR(255) NOT NULL,
+           resource VARCHAR(255),
+           service TEXT,
+           event_name VARCHAR(255),
+           group_name VARCHAR(255),
+           tags TEXT,
+           startTime TIMESTAMP,
+           duration INT,
+           reason TEXT
```

Comment:
mb rename to `text`?

Reply:
in alerta gui showed reason instead of text